crates/common/types/transaction.rs

```diff
@@ -35,6 +37,116 @@ pub enum Transaction {
     PrivilegedL2Transaction(PrivilegedL2Transaction),
 }
 
+/// The same as a Transaction enum, only that blob transactions are in wrapped format, including
+/// the blobs bundle.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum P2PTransaction {
```

Comment:
If this is a structure specifically made to fit the format expected by p2p can we move it there? Placing it in the same module as core types can lead to mix ups in the future